tests/unit/h_vialib/client_test.py

```diff
@@ -0,0 +1,49 @@
+from h_matchers import Any
+
+from h_vialib import ViaClient, ViaDoc
+
+
+class TestViaDoc:
+    def test_no_content_type(self):
+        doc = ViaDoc("http://random.document.com")
+        assert doc.is_pdf == False
+
+    def test_explicit_content_type(self):
+        doc = ViaDoc("http://random.document.com", content_type="pdf")
+        assert doc.is_pdf == True
+
+    def test_url_matches_gdrive_pdf(self):
+        doc = ViaDoc("https://drive.google.com/uc?id=0&export=download")
+        assert doc.is_pdf == True
+
+
+class TestViaClient:
+
+    DEFAULT_OPTIONS = {
+        "via.client.openSidebar": "1",
+        "via.client.requestConfigFromFrame.origin": "http://random.localhost",
+        "via.client.requestConfigFromFrame.ancestorLevel": "2",
+        "via.external_link_mode": "new-tab",
+    }
+    VIA_URL = "http://via.localhost"
+    ORIGIN_URL = "http://random.localhost"
+
+    def test_client_default_options(self):
+        proxied_url = "http://example.com"
+
+        final_url = ViaClient(self.VIA_URL, self.ORIGIN_URL).url_for(
```

Comment:
If this was a fixture, then we'd be able to tuck these values away too (although I see we use the Via URL elsewhere)?

```
@pytest.fixture
def client():
    return ViaClient(self.VIA_URL, "http://random.localhost")
```

Then the test would focus more on the bit we care about (`url_for`) rather than init.